### PR TITLE
Bits complexity takes maximum bitwidth for paired activation and weight

### PR DESCRIPTION
**Pull request by ljaljushkin**
ticket: 50340

### CODE REVIEW

File: nncf/quantization/algo.py [hunk 1]
```diff
@@ -1826,7 +1831,7 @@ def __init__(self, target_model: NNCFNetwork,
                  weight_quantizers: Dict[WeightQuantizerId, WeightQuantizerInfo],
                  non_weight_quantizers: Dict[NonWeightQuantizerId, NonWeightQuantizerInfo],
                  groups_of_adjacent_quantizers: GroupsOfAdjacentQuantizers,
-                 initial_quantizer_setup: SingleConfigQuantizerSetup,
+                 quantizer_setup: SingleConfigQuantizerSetup,
```
Comment (vshampor) [inline nncf/quantization/algo.py:1834]: Shouldn't rename this.

File: nncf/quantization/algo.py [hunk 2]
```diff
@@ -1342,6 +1345,7 @@ def __init__(self, target_model: NNCFNetwork,
                  weight_quantizers: Dict[WeightQuantizerId, WeightQuantizerInfo],
                  non_weight_quantizers: Dict[NonWeightQuantizerId, NonWeightQuantizerInfo],
                  groups_of_adjacent_quantizers: GroupsOfAdjacentQuantizers,
+                 quantizer_setup: SingleConfigQuantizerSetup,
```
Comment (vshampor) [inline nncf/quantization/algo.py:1348]: Shouldn't extend the regular controller with a setup parameter right now. I understand that this might be required in the scope of quantizer setup export, but this should be done in a separate PR.

**Review by vshampor — CHANGES_REQUESTED** [review]

File: nncf/quantization/algo.py (hunk 1 above)
Comment (ljaljushkin) [reply nncf/quantization/algo.py]: ok, reverted

File: nncf/quantization/algo.py (hunk 2 above)
Comment (ljaljushkin) [reply nncf/quantization/algo.py]: ok, reverted

**Review by vshampor — APPROVED** [review]

**Comment by ljaljushkin** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Jenkins please retry a build
